doc/manual/metaprogramming.rst

```diff
@@ -134,19 +162,88 @@ ambiguity in parsing.:
     julia> :(::)
     :(::)
 
-:obj:`Symbol`\ s can also be created using :func:`symbol`, which takes
-a character or string as its argument:
+Expressions and evaluation
+--------------------------
+
+Quoting
+~~~~~~~
+
+There is special syntax to create expression objects without using the
```

Comment:
In your opening paragraph about symbols, you state that 

> The `:` character has two syntactic purposes in Julia. The first form creates a [...]

It would be nice to refer back to this statement, maybe re-wording this as

> The second syntactic purpose of the  `:` character is to construct expression objects without using the explicit `Expr` constructor; this is sometimes referred to as quoting *quoting*."